### PR TITLE
Update harden_server.rst

### PR DESCRIPTION
**Pull request by Rello**
### ☑️ Resolves

* complete url list incl protocol for firewall related question

### CODE REVIEW

**Review by schiessle — APPROVED** [review]
great, thanks a lot!

File: admin_manual/installation/harden_server.rst [hunk 1]
```diff
@@ -240,13 +240,16 @@ Some Nextcloud functionality requires connecting to remote servers. Depending on
...
 
 - www.nextcloud.com, www.startpage.com, www.eff.org, www.edri.org for checking the internet connection
-- apps.nextcloud.com for the available apps
-- updates.nextcloud.com for Nextcloud updates
-- lookup.nextcloud.com For updating and lookup in the federated sharing addressbook
-- push-notifications.nextcloud.com for sending push notifications to mobile clients
-- surveyserver.nextcloud.com if the admin has agreed to share anonymized data
+- cloud.nextcloud.com (https) for validating the enterprise subscription
+- apps.nextcloud.com (https) for available apps and their updates 
+   - github.com (https) for downloading Nextcloud standard apps
```
Comment (szaimen) [inline admin_manual/installation/harden_server.rst:245]:
```suggestion
- github.com (https) for downloading Nextcloud standard apps
```

File: admin_manual/installation/harden_server.rst [hunk 2]
```diff
@@ -240,13 +240,16 @@ Some Nextcloud functionality requires connecting to remote servers. Depending on
...
+- updates.nextcloud.com (https) for Nextcloud updates
+- lookup.nextcloud.com (https) for updating and lookup in the federated sharing addressbook
+- push-notifications.nextcloud.com (https) for sending push notifications to mobile clients
+- surveyserver.nextcloud.com (https) if the admin has agreed to share anonymized data
 - Any remote Nextcloud server that is connected with federated sharing
 
+
```
Comment (szaimen) [inline admin_manual/installation/harden_server.rst:252]:
```suggestion
```

File: admin_manual/installation/harden_server.rst [hunk 3]
```diff
@@ -240,11 +240,13 @@ Some Nextcloud functionality requires connecting to remote servers. Depending on
...
 
 - www.nextcloud.com, www.startpage.com, www.eff.org, www.edri.org for checking the internet connection
-- apps.nextcloud.com for the available apps
-- updates.nextcloud.com for Nextcloud updates
-- lookup.nextcloud.com For updating and lookup in the federated sharing addressbook
-- push-notifications.nextcloud.com for sending push notifications to mobile clients
-- surveyserver.nextcloud.com if the admin has agreed to share anonymized data
+- cloud.nextcloud.com (https) for validating the enterprise subscription
+- apps.nextcloud.com (https) for available apps and their updates 
+- github.com (https) for downloading Nextcloud standard apps
```
Comment (nickvergessen) [inline admin_manual/installation/harden_server.rst:245]: Can we move this to the end, so it's more visible (since it's not our server)

Comment (nickvergessen) [reply admin_manual/installation/harden_server.rst:245]: Also pushfeed.nextcloud.com (https) could be listed:

https://github.com/nextcloud/nextcloud_announcements/blob/master/lib/Cron/Crawler.php#L38C35-L38C57

**Pull request merged**